modular_citadel/code/game/objects/items/melee/eutactic_blades.dm

```diff
@@ -304,7 +304,7 @@
 		var/mob/M = loc
 		M.update_inv_hands()
 
-	SEND_SIGNAL(src, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)//blood overlays get weird otherwise, because the sprite changes. (retained from original desword because I have no idea what this is)
```

Comment:
Comsig

Reply:
ditto with that dsword. it's only for blood overlay fuckery